### ISSUE TITLE
the bert only accept token_ids and mask but segment_ids

### ISSUE THREAD

**Issue by breadbread1984** [kpe/bert-for-tf2#4]
I want to do classification with bert. I found the bert only has two input arguments which are inputs and mask. I need to input segment ids for classification. how do I input this argument? thx

**Comment by mriganktiwari** [comment]
Did you happen to resolve this? I am trying to get "input_ids", "attention_mask" and "segment_ids" from the FullTokenizer, would you know how can I get that?
